specification/databricks/resource-manager/Microsoft.Databricks/stable/2024-05-01/accessconnector.json

```diff
@@ -0,0 +1,438 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Databricks/accessConnectors": {
+      "get": {
+        "tags": [
+          "AccessConnector"
+        ],
+        "operationId": "AccessConnectors_ListBySubscription",
+        "description": "Gets all the azure databricks accessConnectors within a subscription.",
+        "x-ms-examples": {
+          "Lists all the azure databricks accessConnectors within a subscription.": {
+            "$ref": "./examples/AccessConnectorsListBySubscriptionId.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccessConnector": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "systemData": {
+          "description": "The system metadata relating to this resource",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "description": "Azure Databricks accessConnector properties",
+          "$ref": "#/definitions/AccessConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/TrackedResource"
```

Comment:
>       "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/TrackedResource" [](http://example.com/codeflow?start=3&length=94)

Seeing this I feel curious if there's a reason not to inherit common-types types.json directly. I guess you have something or other custom?

Reply:
@TimLovellSmith this was discussed earlier with other reviewers as well, we have some differences w.r.t common and our type, so we didn't change. Our feature work doesn't have scope to address this so we are not able to resuse.

Reply:
If we aren't getting linter errors for it, usually its not a critical difference. It does make reviewing harder though. :-(

Reply:
Agree with it, that is why there is a base commit and newer commits approach is suggested by ARM team I believe. JFYI this PR is already being reviewed and approved. Last night @kazrael2119 removed a suppression which removed a tag.